README.md

```diff
@@ -105,10 +82,13 @@ kubectl get pods
 
 Please read our [CONTRIBUTING](CONTRIBUTING.md) guide before making a pull request. Refer to our [DEVELOPMENT](./ray-operator/DEVELOPMENT.md) to build and run tests locally.
 
-### Getting involved
-Kuberay has an active community of developers. Here’s how to get involved with the Kuberay community:
+## Getting Involved
+
+Join [Ray's Slack workspace](https://docs.google.com/forms/d/e/1FAIpQLSfAcoiLCHOguOm8e7Jnn-JJdZaCxPGjgVCvFijHB5PLaQLeig/viewform), and search the following public channels:
+
+* `#kuberay-questions` (KubeRay users): This channel aims to help KubeRay users with their questions. The messages will be closely monitored by the Ray/KubeRay maintainers.
```

Comment:
```suggestion
* `#kuberay-questions` (KubeRay users): This channel aims to help KubeRay users with their questions. The messages will be closely monitored by the Ray and KubeRay maintainers.
```

Reply:
I tried to create invitation links for the channels, but they will expire in 14 days.

Reply:
Gotcha, this is fine then